### DISCUSSION TITLE
Speaking with manufacturer [GIDERWEL] regarding LED strips

### DISCUSSION THREAD

**Discussion by dbroome1978** [Koenkk/zigbee2mqtt#8051] [Devices]
Good morning,
I recently bought [GIDERWEL Smart ZigBee 32.8ft RGB LED Strip Lights](https://www.amazon.com/gp/product/B093KC4SGG)
I returned it since I could not get it to work - when in debug mode, it didn't even show the model/vendor info.  I think I understand that might be related which endpoint that info is broadcasted on.
The manufacturer reached out to me and I explained I was returning it since it wasn't support here.  I pointed out that Gledopto lights are supported.
They responded with 

> Do you mean that you now have Gledopto branded products compatible with your device? Does Gledopto support •mqtt or •.0? Under normal circumstances, Gledopto compatible devices, our products can also be compatible. We are using the same procedure, can you give me a detailed description of the problem with our light strips? 

I'm honestly impressed they are even having this dialog with me.
My question, how hard would it be for me to troubleshoot?  I have already made a few converters and have no problem trying to make another - especially considering that it should be fairly similar to the converter already made for Gledopto.  

Are we able to try to grab the model/vendor from another endpoint?  If so, I'll repurchase this and try to get it supported for myself and this project.

I don't have any logs and I realize there are no guartentees, but I'm willing to try if the vendor/model issue is something we can workaround.

**Comment by Koenkk** [comment]
Please provide the herdsman debug log when pairing the device.

See https://www.zigbee2mqtt.io/information/debug.html#zigbee-herdsman-debug-logging on how to enable the herdsman debug logging. Note that this is only logged to STDOUT and not to log files.

**Comment by dbroome1978** [comment]
Ok - I'll reorder it and test it out.

You and I troubleshooted to get the Moes 4gang switch working.  I tried to
debug for this one but couldn't even add a converter due to the device
reporting very generic information "Zigbee model 'A10' and manufacturer
name 'Light' is NOT supported"

Once it paired, it had more endpoints listed but was quickly beyond my
range.

Any pointers I can start reviewing once it arrives?  I'll attach the full
stdout info once I have it

On Thu, Jul 15, 2021 at 11:43 AM Koen Kanters ***@***.***>
wrote:

> Please provide the herdsman debug log when pairing the device.
>
> See
> https://www.zigbee2mqtt.io/information/debug.html#zigbee-herdsman-debug-logging
> on how to enable the herdsman debug logging. Note that this is only logged
> to STDOUT and not to log files.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Koenkk/zigbee2mqtt/discussions/8051#discussioncomment-1009548>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHCR6IOUF65Q2S5BBIRXPS3TX4F4RANCNFSM5AN4XUOQ>
> .
>


-- 
David

**Reply by Koenkk** [reply]
Hard to do anything without logs, lets wait for the logs
